{{ cookiecutter.package_name }}/pyproject.toml

```
@@ -87,3 +88,63 @@ write_to = "{{ cookiecutter.module_name }}/_version.py"
 {%- else -%}
 write_to = "{{ cookiecutter.module_name }}/version.py"
 {%- endif %}
+
+[tool.gilesbot]
+  [tool.gilesbot.pull_requests]
+  enabled = true
+
+  [tool.gilesbot.towncrier_changelog]
+  enabled = true
+  verify_pr_number = true
+  changelog_skip_label = "No Changelog Entry Needed"
+  help_url = "https://github.com/{{cookiecutter.github_repo}}/blob/main/changelog/README.rst"
```

This is just recommended jinja style:

```suggestion
  help_url = "https://github.com/{{ cookiecutter.github_repo }}/blob/main/changelog/README.rst"
```